ert/data/record/_record.py

```diff
@@ -59,6 +59,14 @@ class RecordType(str, Enum):
 
 
 class Record(ABC):
+    """The :class:`Record` class is an abstract class that
+    includes record data and the record type. It represents a
+    basic data `unit` to accommodate input and output to stages.
+
+    The record needs be either :class:`BlobRecord` or
+    :class:`NumericalRecord` type derived from `Record` class.
```

Comment:
Maybe a bit weird to document sub-classes in the base class? Could probably just remove this?

Reply:
It's still part of the official documentation hence I would keep it as it provides at least links to the derived sub-classes.